main/java/sorts/BubbleSort.java

```diff
@@ -0,0 +1,94 @@
+package sorts;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class BubbleSort implements Sort {
+
+	@Override
+	public void sortArrayFirstApproach(double[] list) {		//this is close to n^2 complexity always because it swaps always
+		int n = list.length;
+		boolean sorted;	
+		double temp;
+		
+		do {
+			sorted = false;
+			for (int i = 0; i < n - 1; i++) {
+				if (list[i] > list[i + 1]) {
+					sorted = true;
+					
+					temp = list[i];
+					list[i] = list[i + 1];
+					list[i + 1] = temp;
+				}
+			}
+			
+		} while (sorted);
+	}
+
+	@Override
+	public void sortArraySecondApproach(double[] list) {	// in best case n - complexity
```

Comment:
Be more specific about naming for both functions.. First Approach, Second approach is not acceptable
Also extract implementations into 2 separate classes.